src/Text/Parser/TreeSitter/Language.hs

```diff
@@ -32,17 +32,24 @@ class Symbol s where
 -- | TemplateHaskell construction of a datatype for the referenced Language.
 mkSymbolDatatype :: Name -> Ptr Language -> Q [Dec]
 mkSymbolDatatype name language = do
-  symbols <- runIO $ languageSymbols language
+  symbols <- ((Regular, "ParseError") :) <$> runIO (languageSymbols language)
```

Comment:
We prepend a hard-coded `ParseError` constructor onto the list of symbols to ensure consistency across the defined instances &c.